### ISSUE TITLE
A10, Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by stevensu1977** [lllyasviel/Fooocus#823]
**Describe the problem**
A10,  8Core, 32GB, Python 3.10.13,  always got Segmentation fault  error .

**Full Console Log**
(fooocus) ubuntu@ip-172-31-50-239:~/Fooocus$ python entry_with_update.py --listen
Already up-to-date
Update succeeded.
Python 3.10.13 (main, Sep 11 2023, 13:44:35) [GCC 11.2.0]
Fooocus version: 2.1.769
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
Total VRAM 22564 MB, total RAM 31739 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA A10G : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
[Fooocus] Disabling smart memory
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
missing {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: /home/ubuntu/Fooocus/models/checkpoints/juggernautXL_version6Rundiffusion.safetensors
LoRAs loaded: [('sd_xl_offset_example-lora_1.0.safetensors', 0.1), ('None', 0.1), ('None', 0.1), ('None', 0.1), ('None', 0.1)]
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.25 seconds
App started successful. Use the app with http://localhost:7860/ or 0.0.0.0:7860
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 7990070537218009957
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 24
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] fish, intricate, elegant, glowing light, sharp focus, fair quality, dynamic light, epic composition, dramatic cinematic, extremely detailed, detailed, cinematic light, cute, magical, stunning,, noble, brilliant, brilliant, light glowing, striking, wonderful, pretty, perfect, artistic, extremely detailed, fantastic, pure, brilliant, brilliant, perfect, perfect, colorful
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] fish, intricate, elegant, colorful, vivid, detailed, extremely detailed, detailed, cinematic, surreal, beautiful, pure glowing, attractive, intricate, beautiful,led, shiny, bright glowing, bright, colorful, surreal, pretty, cosmic background, best, romantic, inspired, inspired, beautiful, inspired, deep detailed, intricate detailed, lovely, intricate, cool, awesome
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
Preparation time: 2.69 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.02916753850877285, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.24 seconds
  0%|                                                                                   | 0/30 [00:00<?, ?it/s]Segmentation fault (core dumped)

**Comment by stevensu1977** [comment]
I'm missing some package , after I install CLIP 1.0 , it's work fine.

**Comment by LukeLIN-web** [comment]
I meet the  same problem.
